src/sentry/integrations/slack/message_builder/image_block_builder.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+import sentry_sdk
+
+from sentry import features
+from sentry.api import client
+from sentry.charts import backend as charts
+from sentry.charts.types import ChartType
+from sentry.integrations.slack.message_builder import SlackBlock
+from sentry.integrations.slack.message_builder.base.block import BlockSlackMessageBuilder
+from sentry.integrations.slack.message_builder.time_utils import (
+    get_approx_start_time,
+    get_relative_time,
+)
+from sentry.issues.grouptype import PerformanceP95EndpointRegressionGroupType
+from sentry.models.apikey import ApiKey
+from sentry.models.group import Group
+from sentry.snuba.referrer import Referrer
+from sentry.utils.performance_issues.detectors.utils import escape_transaction
+
+logger = logging.getLogger("sentry.integrations.slack")
+
+
+class ImageBlockBuilder(BlockSlackMessageBuilder):
+    def __init__(self, group: Group) -> None:
+        super().__init__()
+        self.group = group
+        self.event = group.get_latest_event_for_environments()
+
+    def build_image_block(self) -> SlackBlock | None:
+        if (
+            features.has("organizations:slack-endpoint-regression-image", self.group.organization)
+            and self.group.issue_type == PerformanceP95EndpointRegressionGroupType
+        ):
+            return self._build_endpoint_regression_image_block()
+
+        # TODO: Add support for other issue alerts
+        logger.warning("build_issue_alert_image.not_supported")
+        return None
+
+    def _build_endpoint_regression_image_block(self) -> SlackBlock | None:
+        logger.info(
+            "build_endpoint_regression_image.attempt",
+            extra={
+                "group_id": self.group.id,
+            },
+        )
+
+        organization = self.group.organization
+        if self.event is None or self.event.transaction is None or self.event.occurrence is None:
+            return None
+        transaction_name = escape_transaction(self.event.transaction)
+        period = get_relative_time(anchor=get_approx_start_time(self.group), relative_days=14)
```

Comment:
The frontend uses relative_days set to 7 days
```suggestion
        period = get_relative_time(anchor=get_approx_start_time(self.group), relative_days=7)
```

Reply:
Discussed offline. 14 is fine to match this logic: https://github.com/getsentry/sentry/blob/master/static/app/components/events/eventStatisticalDetector/breakpointChart.tsx#L16